template/src/__tests__/App-test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import App from '../App';
+import renderer from 'react/lib/ReactTestRenderer';
+
+describe('App', () => {
+  it('renders a welcome view', () => {
+    const instance = renderer.create(<App />);
+    const tree = instance.toJSON();
```

Comment:
We are likely to add more APIs to the instance to do selections (similar to enzyme, except component based, like `instance.find(Button)`). Right now people shouldn't do more than `instance.toJSON()` and `instance.props.on*()` calls. Everything else is subject to change.